experiment/run-library.sh

```diff
@@ -572,34 +592,74 @@ function a_out() {
     # Just run the ansible command. No output processing, action logging or measurements
     if $run_lib_dryrun; then
         echo 'FAKE ansible RUN'
-    else
```

Comment:
It this intentional? If so, maybe we can remove this if here.

Reply:
I think this one sneak in :/ Thanks!!